docs/modules/administration/pages/update-2-to-3.adoc

```diff
@@ -0,0 +1,137 @@
+== Update your OpenDevStack repositories
+
+Updating repositories means that new refs from repositories under
+`github.com/opendevstack` are pushed into the repositories in your BitBucket
+instance.
+
+First, you need a clone of each repository in BitBucket which should be updated
+on your local machine.
+
+Once this has been done, you need to fetch new refs from
+`github.com/opendevstack`. To do so, add a remote pointing to it like this:
+
+[source,sh]
+----
+git remote add ods https://github.com/opendevstack/<REPO_NAME>.git
+----
+
+Now you are ready to update the refs. It is recommended to update both the
+`master` branch and, unless you want to live off the bleeding edge, a release
+branch such as `2.x`. Use the steps shown below:
+
+[source,sh]
+----
+# Ensure you have the latest refs from ODS locally
+git fetch ods
+# Update master
+git checkout master
+git reset --hard ods/master
+git push origin master
+# Update 2.x
+git checkout 2.x
+git reset --hard ods/2.x
+git push origin 2.x
+----
+
+If your OpenDevStack installation is based on a custom branch (such as `2.acme`), then you
+need to create a pull request on BitBucket from `2.x` into that custom branch now.
+
+Now that the repositories are updated, you also need to modify the images and the
+running instances in OpenShift.
+
+== Backup
+
+Before proceeding, it is advisable to make a backup of the existing OpenShift
+configuration. This can be done easily with Tailor:
+
+[source,sh]
+----
+# Backup CD project
+tailor export -n cd > backup_CD.yml
+
+# Backup provision app namespaces
+tailor export -n prov-cd > backup_PROV_CD.yml
+tailor export -n prov-dev > backup_PROV_DEV.yml
+tailor export -n prov-test > backup_PROV_TEST.yml
+----
+
+Note that the executing user needs to have permissions to access all resources
+in the `cd` namespaces for this to work properly.
+
+== Tailor
+
+Next, update Tailor.
+3.x requires Tailor https://github.com/opendevstack/tailor/releases/tag/v1.1.1[1.1.1].
+
+== Configuration
+
+Then, update/add/remove the configuration parameters (located in `ods-configuration`).
+To do this, use the `./update` script located in `ods-core/configuration-sample`.
+
+== OCP resources
+
+Next, run `tailor update` in `ods-core` and `ods-quickstarters` to bring all OCP resources (such as DC or BC) into sync. Review the diff produced by Tailor carefully, especially around changes to PVCs.
```

Comment:
Please update to `tailor apply` instead of `tailor update`. However, people should probably run the `Makefile` targets from the installation section.